packages/graphql-extensions/src/index.ts

```diff
@@ -99,6 +103,17 @@ export class GraphQLExtensionStack<TContext = any> {
       ext => ext.parsingDidStart && ext.parsingDidStart(o),
     );
   }
+  public setParsedDocumentAndOperationName(
```

Comment:
I don't think the name of this method name really needs to have `parsed` in it.

Reply:
Sure! `setOperationDocument`?

Reply:
Maybe just `setDocumentAndOperationName` if we need to keep it?